src/IECoreRI/RendererImplementation.cpp

```diff
@@ -1823,12 +1824,69 @@ void IECoreRI::RendererImplementation::emitCurvesPrimitive( const IECore::Curves
 				(char *)( primitive->periodic() ? "periodic" : "nonperiodic" ),
 				pv.n(), pv.tokens(), pv.values() );
 }
-	
+
+namespace
+{
+
+// note this is duplicated from src/IECore/Primitive.cpp
+// perhaps it should be centralized, either publicly, or
+// via the private headers (e.g. PrimitiveAlgoUtils.h)
+void convertUVs( PrimitiveVariableMap &variables )
+{
+	std::set<Data *> visited;
+
+	auto tIt = variables.find( "t" );
+	if( tIt != variables.end() )
+	{
+		if( FloatVectorData *values = runTimeCast<FloatVectorData>( tIt->second.data.get() ) )
+		{
+			for( auto &value : values->writable() )
+			{
+				value = 1.0f - value;
+			}
+		}
+
+		visited.insert( tIt->second.data.get() );
+	}
+
+	// by convention, PrimitiveVariables ending with "_t"
+	// represent the second component of a UV set.
+	for( auto it = variables.begin(), eIt = variables.end(); it != eIt; ++it )
+	{
+		if( visited.find( it->second.data.get() ) != visited.end() )
+		{
+			continue;
+		}
+
+		size_t suffixOffset = it->first.rfind( "_t" );
```

Comment:
Same comment as for Primitive.cpp - it'd be good to avoid the duplicate logic, and use ends_with.

Reply:
Addressed in b4ea35d